diff --git a/tools/msvs_env.bat b/tools/msvs_env.bat
index ac44e293..e2f0b1c4 100644
--- a/tools/msvs_env.bat
+++ b/tools/msvs_env.bat
@@ -37,6 +37,14 @@ for %%x in ("%PROGRAMFILES(X86)%" "%PROGRAMFILES%") do (
   )
 )
 
+:: Search for the default VS2019 installation path.
+for %%x in ("%PROGRAMFILES(X86)%" "%PROGRAMFILES%") do (
+  for %%y in (Professional Enterprise Community BuildTools) do (
+    set vcvars="%%~x\Microsoft Visual Studio\2019\%%y\VC\Auxiliary\Build\%vcvarsbat%"
+    if exist !vcvars! goto found_vcvars
+  )
+)
+
 echo ERROR: Failed to find vcvars
 set ERRORLEVEL=1
 goto end
